official/recommendation/ncf_common.py

```diff
@@ -109,13 +109,15 @@ def parse_flags(flags_obj):
   }
 
 
-def get_optimizer(params):
-  optimizer = tf.train.AdamOptimizer(
+def get_keras_optimizer(params):
```

Comment:
do you need a helper method if this is used in only one place?

Reply:
There is still an if logic in the method, so I think we could keep the method. 
But it is used in keras, so I move it out of ncf_common to ncf_keras_main.

Reply:
the else part seems irrelevant for the keras case anyway - it is for TPUEstimator. so i dont see why the method is helpful in your codepath